src/test/starknet.t.sol

```diff
@@ -32,6 +34,7 @@ contract ConfigStarknet {
     function setValues() public {
         starknetValues = StarknetValues({
             core_implementation:       0x60C5fA1763cC9CB9c7c25458C6cDDFbc8F125256,
+            l2_teleport_gateway:       0x03a85abf730fb56410c92841a4439efcf24a2efe0085fb2e7807f0a6f48a1b39,
```

Comment:
not sure about this yet, but eventually a check should be included below (e.g. `checkStarknetTeleport) eventually, together with other key values to monitor.